src/components/shared/searchView/SearchResults.tsx

```diff
@@ -176,6 +142,7 @@ class SearchResults extends React.Component<ISearchResultsProps, ISearchResultsS
                 </div>
             );
         }
+        const isRoutesView = Navigator.getPathName() === subSites.routes;
```

Comment:
Would be safer to use some router matching logic instead of just comparing the values